crates/db/src/errors.rs

```diff
@@ -34,6 +34,12 @@ pub enum DbError {
     #[error("Block does not extend canonical chain tip")]
     L1InvalidNextBlock(u64, L1BlockId),
 
+    #[error("missing L2 block (idx {0})")]
+    MissingL2Block(L2BlockId),
+
+    #[error("missing L2 block (idx {0})")]
+    MissingL2BlockHeight(u64),
```

Comment:
First one mistakenly says `idx` instead of `id`.

And the second one should be defined in terms of "slot".